Makefile

```diff
@@ -1,71 +1,86 @@
 # System Setup
 SHELL = bash
 
-RUST_VERSION ?= stable
```

Comment:
This could be removed as it was never actually expanded within make. Instead, `RUST_VERSION` is optional and can be provided when calling make, like `RUST_VERSION=beta make build`.